src/lib/bodyParsing/fileParser.ts

```diff
@@ -6,8 +6,8 @@ declare module 'express-serve-static-core' {
     }
 }
 
-export default function fileParser() {
+export default function fileParser(options: multer.Options = {}) {
     const storage = multer.memoryStorage();
-    const fileUpload = multer({ storage });
+    const fileUpload = multer({ ...options, storage });
```

Comment:
memory storage is the one we supply by default, we should not overwrite it if its passed in options

Reply:
i switched options and storage